lib/utils/token/getUserProfile.ts

```
@@ -1,4 +1,4 @@
-import { getDecodedToken } from ".";
+import { getClaims, getDecodedToken } from ".";
```

ESLint

[error] 'getDecodedToken' is defined but never used.

(@typescript-eslint/no-unused-vars)